app/views/templates/rulingCertificateTemplate.scala.html

```diff
@@ -20,33 +20,28 @@
 @(appConfig: FrontendAppConfig, c: Case, d: Decision, getCountryName: String => Option[String],
 compositeMode: Boolean = false)(implicit messages: Messages)
 
-@createLink(path: String)(implicit req: play.api.mvc.Request[_]) = @{
-val protocol = if(req.secure) "https" else "http"
-s"$protocol://${req.host}${routes.Assets.versioned(path)}"
-}
-
-<div id="print-pages" class="print-pages">
-    <div id="ruling-certificate" class="print-document" id="print-document">
+<div class="print-pages">
+    <div class="print-document" id="print-document">
         <div class="govuk-grid-row print-header">
             <div class="govuk-grid-column-one-third">
                 <img class="crown-logo-atar" alt="HM Revenue and Customs" src="@messages("logo.encoded")"/>
             </div>
             <div>
-                <h2 class="govuk-heading-m margin-top-small no-margin-bottom print-header__heading">
+                <h1 class="govuk-heading-m margin-top-small no-margin-bottom print-header__heading">
```

Comment:
add "green-ruling-application" as a class

Reply:
Similar issue as the application letter view:

<img width="1340" alt="Screenshot 2023-09-07 at 14 56 59" src="https://github.com/hmrc/binding-tariff-trader-frontend/assets/47248104/bd4dc8fb-a7ac-460f-8e07-89efc0afb13c">

<img width="1366" alt="Screenshot 2023-09-07 at 14 57 18" src="https://github.com/hmrc/binding-tariff-trader-frontend/assets/47248104/e8712603-2bfc-433e-b4d7-36bc318ca0b2">

Reply:
With new css:

<img width="1447" alt="Screenshot 2023-09-07 at 14 58 02" src="https://github.com/hmrc/binding-tariff-trader-frontend/assets/47248104/dc9c6142-b1f7-4397-b1dc-c9ca6c5f96a5">

<img width="1365" alt="Screenshot 2023-09-07 at 14 58 10" src="https://github.com/hmrc/binding-tariff-trader-frontend/assets/47248104/fed59331-ec86-4aa5-8c33-5255b4db22bb">